### DISCUSSION TITLE
Entity description

### DISCUSSION THREAD

**Discussion by disforw** [kdeyev/eyeonwater#52] [General]
I hope you don’t mind, I removed the name and changed the UID to use the key. As you already mentioned, this makes it breaking change, but I feel like it’s better to break it now rather than later.
Also, HA handles battery sensor naming automatically, so there are no translations for those. I had to make the translation_key in the dataclass optional. Just a heads up, pretty soon the tampers will be named automatically as well, so those will come out too.

**Comment by kdeyev** [comment]
I don't mind at all. It makes a lot of sense and I'm looking forward to these changes.
